server/backends/blobstore/blobstore.go

```diff
@@ -287,7 +305,9 @@ func (g *GCSBlobStore) ReadBlob(ctx context.Context, blobName string) ([]byte, e
 		return nil, err
 	}
 	start := time.Now()
+	_, spn := tracing.StartSpan(ctx)
```

Comment:
nit: i know ctx isn't used here, but does the compiler complain if you do

"ctx, spn := tracing.StartSpan(ctx)"?

It feels like that might be less error-prone going forward, if someone does introduce code that takes a context, they won't need to also know that _ is a ctx we are ignoring and change that too.

Reply:
Nah, the compiler is cool with it since we aren't declaring ctx, only assigning to it. I think that makes sense - swapped it over.